pkgs/data/fonts/league-of-moveable-type/default.nix

```diff
@@ -23,6 +23,8 @@ stdenv.mkDerivation rec {
   installPhase = ''
     mkdir -p $out/share/fonts/truetype
     cp */*.otf $out/share/fonts/truetype
+    # for Raleway, where the fonts are already in /share/…
+    cp */share/fonts/truetype/*.otf $out/share/fonts/truetype
```

Comment:
Hmm, it is a bit odd that the OTF files are placed in `$out/share/fonts/truetype` in this package. Conventionally they should be in `$out/share/fonts/opentype`.